tools/generate-bolts.py

```diff
@@ -238,13 +238,16 @@ def true_type(type_name, field_name=None):
 
     def __init__(self, name):
         FieldSet.__init__(self)
-        self.name = name
+        self.name, self.is_enum = self.parse_name(name)
         self.depends_on = {}
-        # FIXME: internal msgs can be enums
-        self.is_enum = False
         self.type_comments = []
         self.tlv = False
 
+    def parse_name(self, name):
+        if name.startswith('e:'):
+            return name[2:], True
```

Comment:
I prefer 'enum ' here over 'e:'.  But it's a trivial change, so can happen afterwards.

Reply:
ok, i'm not partial to either. put in a patch commit at the end that makes this change!